lib/core/src/Cardano/Wallet/Transaction.hs

```diff
@@ -143,6 +146,15 @@ data TransactionLayer k tx = TransactionLayer
         -> Coin
         -- ^ Compute an execution costs of scripts in a given transaction.
 
+    , evaluateMinimumFee
+        :: Node.ProtocolParameters
+            -- Current protocol parameters
+        -> tx
+            -- The sealed transaction
+        -> Coin
```

Comment:
Perhaps we should return `Maybe` instead of erroring in case of byron, but I think leaving it as-is is fine while still trying to fit the pieces together.

Reply:
I ended up adding a `Maybe` here.